<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>400</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>CCAligner_GUI</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="ActivityTextbox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>371</width>
      <height>311</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ActivityLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ACTIVITY</string>
    </property>
   </widget>
   <widget class="QLabel" name="InputWavLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT .WAV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="WavOpenButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File...</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChosenWavLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No input .wav file chosen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="InputSrtLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INPUT .SRT</string>
    </property>
   </widget>
   <widget class="QLabel" name="HeaderLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-10</y>
      <width>311</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chiller</family>
      <pointsize>45</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CCAligner GUI</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SrtOpenButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File...</string>
    </property>
   </widget>
   <widget class="QLabel" name="ChosenSrtLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>No input .srt file chosen</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
